flake.nix

```
@@ -7,6 +7,14 @@
 
   outputs = { self, nixpkgs }: {
 
+    nixosModules.default = {...}: {
```

```suggestion
    nixosModules.garnix = {...}: {
```
I don't think it matters that much whether it's `default` or `garnix`. But it should be one module for both this and the `persistence` config from #1.